types/account.go

```diff
@@ -3,6 +3,8 @@ package types
 import (
 	"fmt"
 
+	"github.com/ethereum/go-ethereum/log"
```

Comment:
probably shouldn't be mixing loggers with geths, maybe should just panic inside the conditional below since it is expected that will never happen?

Reply:
Good point. I suppose `using err = fmt.Errorf("error msg %e", err)` would suffice? It seems to be what they are using?